libs/components/core/src/lib/modules/affix/affixer.ts

```diff
@@ -211,7 +211,10 @@ export class SkyAffixer {
   }
 
   #getOffsetParentRect(): AffixRect {
-    if (this.#affixedElement.offsetParent) {
+    if (
+      this.#config.position === 'absolute' &&
+      this.#affixedElement.offsetParent
+    ) {
```

Comment:
There was an assumption that `offsetParent` would only be set for `position: absolute` elements, but Firefox sets the `offsetParent` to `document.body` for `position: fixed` elements.

Reply:
Needs a unit test. Also, wouldn't hurt to put a comment above this condition in the source code, similar to what you have here.

Reply:
Good call. Added a test.